spa/src/components/donationPage/FinishPaymentModal/FinishPaymentModal.tsx

```diff
@@ -0,0 +1,58 @@
+import { ChevronLeft } from '@material-ui/icons';
+import PropTypes, { InferProps } from 'prop-types';
+import { Provider as AlertProvider } from 'react-alert';
+import { Modal } from 'components/base';
+import { StripePaymentWrapper } from 'components/paymentProviders/stripe';
+import Alert, { alertOptions } from 'elements/alert/Alert';
+import { Payment } from 'hooks/usePayment';
+import ContributionDisclaimer from './ContributionDisclaimer';
+import StripePaymentForm from './StripePaymentForm';
+import { BackButton, Root } from './FinishPaymentModal.styled';
+
+const FinishPaymentModalPropTypes = {
+  onCancel: PropTypes.func.isRequired,
+  onError: PropTypes.func,
+  open: PropTypes.bool,
+  payment: PropTypes.object.isRequired
+};
+
+export interface FinishPaymentModalProps extends InferProps<typeof FinishPaymentModalPropTypes> {
+  onCancel: () => void;
+  onError: (error: Error) => void;
+  payment: Payment;
+}
+
+/**
+ * This modal shows a Stripe Payment element that allows the user to complete
+ * their contribution. When they do that, they will be redirected to the
+ * appropriate thank you page.
+ * @see https://stripe.com/docs/payments/payment-element
+ */
+export function FinishPaymentModal({ onCancel, onError, open, payment }: FinishPaymentModalProps) {
```

Comment:
Add default? Or change `open` to be required?

```suggestion
export function FinishPaymentModal({ onCancel, onError, open = false, payment }: FinishPaymentModalProps) {
```

Reply:
My intent was to make it parallel the MUI modal open prop--looks like that is required so I made it the same in 6ec1a969fb3e88d5669ef9a50a6f0ad5688ec062.